go.mod

```diff
@@ -1,96 +1,95 @@
 module github.com/uw-labs/substrate
 
-go 1.18
+go 1.19
 
 require (
-	github.com/Shopify/sarama v1.29.1
+	github.com/Shopify/sarama v1.38.1
 	github.com/Shopify/toxiproxy v2.1.4+incompatible
 	github.com/docker/go-connections v0.4.0
 	github.com/google/uuid v1.3.0
 	github.com/hashicorp/go-multierror v1.1.1
 	github.com/nats-io/nats-streaming-server v0.24.3
-	github.com/nats-io/nats.go v1.13.1-0.20220404202632-8af932f2076b
-	github.com/nats-io/stan.go v0.10.2
-	github.com/prometheus/client_golang v1.12.1
-	github.com/prometheus/client_model v0.2.0
-	github.com/stretchr/testify v1.7.1
-	github.com/testcontainers/testcontainers-go v0.13.0
+	github.com/nats-io/nats.go v1.23.0
+	github.com/nats-io/stan.go v0.10.4
+	github.com/prometheus/client_golang v1.14.0
+	github.com/prometheus/client_model v0.3.0
+	github.com/stretchr/testify v1.8.1
+	github.com/testcontainers/testcontainers-go v0.17.0
```

Comment:
Seems we do need this replace:
```
replace (
    github.com/docker/docker => github.com/docker/docker v20.10.3-0.20221013203545-33ab36d6b304+incompatible // 22.06 branch
)
```
recommended in here: https://golang.testcontainers.org/quickstart/#2-install-testcontainers-for-go
For me, without it, the containers don't manage to start with colima

Reply:
Did you see any errors stopping the containers starting? The pipeline seems to start the containers correctly, and I didn't encounter any issues using Rancher

Reply:
Without it, the containers didn't start

Reply:
These are the logs:
```
2023/01/30 15:28:36 github.com/testcontainers/testcontainers-go - Connected to docker: 
  Server Version: 20.10.20
  API Version: 1.41
  Operating System: Alpine Linux v3.16
  Total Memory: 1976 MB
2023/01/30 15:28:36 Starting container id: 01238076a8a6 image: docker.io/testcontainers/ryuk:0.3.4
2023/01/30 15:28:37 Waiting for container id 01238076a8a6 image: docker.io/testcontainers/ryuk:0.3.4
(1) [] Error response from daemon: No such container: 01238076a8a69defb0e7852870cec18cb97e1a35f0c9762b7bd47f9d8ba9cc6d
(2) [] Error response from daemon: No such container: 01238076a8a69defb0e7852870cec18cb97e1a35f0c9762b7bd47f9d8ba9cc6d

```

Reply:
Yeah I'll just add the replace in, hopefully that ensures compatibility for everyone